docs/guide/tutorial-8-using-ros.md

```diff
@@ -19,7 +19,7 @@ To install the latest version of ROS on Ubuntu use the following commands:
 sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key -o /usr/share/keyrings/ros-archive-keyring.gpg
 sudo apt-get update
-sudo apt-get install ros-noetic-desktop-full # takes time, get a coffee :)
+sudo apt-get install ros-noetic-desktop-full ros-noetic-moveit # takes time, get a coffee :)
```

Comment:
Just note that for compiling `webots_ros` it is enough to install `ros-noetic-moveit-ros-planning-interface`.

Reply:
Additional packages may be required to run specific examples.
I think that in any case these specific packages should then be documented in the ROS tutorials page.

Reply:
Yes, but running the example may fail without the full installation.

Reply:
It is ok.
But note that other packages are also missing to run specific examples, like for example gmapping and probably also others.
So this snippet is not sufficient in any case.